cypress/e2e/API/signIn.spec.js

```diff
@@ -0,0 +1,90 @@
+/// <reference types="cypress" />
+
+import homePage from "../pages/homePage";
+
+describe("SignIn API testing", ()=> {
+    beforeEach(() => {
+        homePage.visit();
+      });
+
+      it("should return userCredentials and ProfileData after signIn", ()=> {
+        const apiUrl = Cypress.env("api_server");
+        cy.intercept("POST", `${apiUrl}/api/auth/signin`).as("SignIn");
+
+        cy.request({
+            method: "POST",
+            url: `${apiUrl}/api/auth/signin`,
+            body: {
+              email: "iryna@gmail.com",
+              password: "asdfghjkl",
```

Comment:
if you are using Cypress variables, you could move secrets like password to env variables too